tests/ops/functions/test_bind_new_parameters.py

```diff
@@ -79,6 +80,11 @@
                 qml.sum(qml.GellMann(1, 3), qml.THermitian(GELL_MANN[6], 1)),
             ),
         ),
+        (
+            qml.ops.LinearCombination([2.0, 3.0], [qml.Hermitian(np.eye(2), 0), qml.X(0)]),
+            (1.0, -np.eye(2), -1.0),
+            qml.ops.LinearCombination([1.0, -1.0], [qml.Hermitian(-np.eye(2), 0), qml.X(0)]),
```

Comment:
Can you also add a case with `Projector`, as well as cases with `Projector` and `Hermitian` inside a `Prod`?

Reply:
Also it seems like the same thing is being tested twice, once in this test, and one in the new test you added. Probably a good idea to remove these test cases and just keep `test_linear_combination`.